### PR TITLE
feat: Styling update

### PR DESCRIPTION
**Pull request by fsbraun**
## Description

This PR fixes some styling issues:

* Update to Node 16 for build process
* Two skins: Traditional and django CMS 4
* Rely on FontAwesome 4 (as it does before) if used with Django, django CMS v3
* Use new icons together with django CMS 4
* Correct order of path in folder drop down
* Replace raw id fields by autocomplete fields in admin
* Add left arrow to select file (to date only visible with djangocms-admin-style)
* Align margins to avoid parts of the admin being hidden by scroll bars
* Only offer canonical URL action button if a canonical URL exists

Also, it recognizes "webp" images as such (for upload - easy thumbnail still creates jpg and png thumbnails).

This PR is thought to iron out some usage quirks for the release of django-filer 3.0

## Related resources

This PR now contains a django CMS v4 icon set in addition to the fontawesome icon set for pure Django and django CMS v3.

## Checklist

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-filer/blob/master/docs/development.rst#contributing) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Review by vinitkumar — APPROVED** [review]
Looks good to me. Let's get this merged and released 🚢  🚀

File: filer/templates/admin/filer/tools/detail_info.html [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
                 {% if original.file.exists %}
-                    <a href="{{ original.url }}" target="_blank" class="btn" download="{{ original.original_filename }}">
-                        <span class="icon fa fa-download"></span>
+                    <a href="{{ original.url }}" target="_blank" class="button" download="{{ original.original_filename }}">
+                        <span class="icon fa fa-download cms-icon cms-icon-download"></span>
                         &nbsp;<span>{% trans "Download" %}</span>
                     </a>
                     {% if original.mime_maintype == 'image' %}
-                    <a href="{{ original.url }}" target="_blank" class="btn float-right">
+                    <a href="{{ original.url }}" target="_blank" class="button">
```
[the remark this answers is not in the document]
Comment (vinitkumar) [reply filer/templates/admin/filer/tools/detail_info.html:14]:
```suggestion
                    <a href="{{ original.url }}" target="_blank" rel="noopener noreferrer" class="button">
```

File: filer/templates/admin/filer/tools/detail_info.html [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
+        <div class="image-info-title"></div>
         <div class="image-details-left">
             <div class="image-preview-container">
                 {% file_icon original detail=True %}
-                <div class="text-left">
+                <div class="text-left button-group">
                 {% if original.file.exists %}
-                    <a href="{{ original.url }}" target="_blank" class="btn" download="{{ original.original_filename }}">
-                        <span class="icon fa fa-download"></span>
+                    <a href="{{ original.url }}" target="_blank" class="button" download="{{ original.original_filename }}">
```
[the remark this answers is not in the document]
Comment (vinitkumar) [reply filer/templates/admin/filer/tools/detail_info.html:9]:
```suggestion
                    <a href="{{ original.url }}" target="_blank" class="button" download="{{ original.original_filename }}" rel="noopener noreferrer">
```

File: filer/templates/admin/filer/folder/directory_table_list.html [hunk 3]
```diff
@@ -138,34 +138,36 @@
                                 <span class="tiny"> ({{ file.size|filesize:"auto1000long" }}{% if file.file_type == "Image" and file.width > 0.0 and file.height > 0.0 %}, {{ file.width|floatformat }}x{{ file.height|floatformat }} px{% endif %})</span>
                             </td>
                             <td class="column-action">
-                                <a href="{{ file.canonical_url }}"
-                                    title="{% blocktrans with file.label as item_label %}Canonical url '{{ item_label }}'{% endblocktrans %}" class="action-button" target="_blank"><span class="fa fa-link"></span></a>
+                                {% if file.canonical_url %}
+                                    <a href="{{ file.canonical_url }}"
```
[the remark this answers is not in the document]
Comment (vinitkumar) [reply filer/templates/admin/filer/folder/directory_table_list.html:142]:
```suggestion
                                    <a href="{{ file.canonical_url }}" rel="noopener noreferrer"
```

**Review by vinitkumar — COMMENTED** [review]
Added some commit suggestions. Please take a look?

File: filer/private/sass/components/_drag-and-drop.scss [hunk 4]
```diff
@@ -358,8 +366,8 @@ form .form-row {
...
...e/svg+xml%3C/dc:format%3E%3Cdc:type rdf:resource='http://purl.org/dc/dcmitype/StillImage' /%3E%3Cdc:title%3E%3C/dc:title%3E%3C/cc:Work%3E%3C/rdf:RDF%3E%3C/metadata%3E%3Cdefs id='defs26' /%3E%3Csodipodi:namedview inkscape:current-layer='svg22' inkscape:window-maximized='0' inkscape:window-y='28' inkscape:window-x='0' inkscape:cy='7.820029' inkscape:cx='12' inkscape:zoom='28.708333' showgrid='false' id='namedview24' inkscape:window-height='776' inkscape:window-width='1440' inkscape:pageshadow='2' inkscape:pageopacity='0' guidetolerance='10' gridtolerance='10' objecttolerance='10' borderopacity='1' inkscape:document-rotation='0' bordercolor='%23666666' pagecolor='%23ffffff' /%3E%3Cg id='g56'%3E%3Cpath inkscape:connector-curvature='0' id='path10' fill='%232980b9' d='M 5,2 C 3.8954,2 3,2.9 3,4 v 8 4 6 c 0,1.1 0.8954,2 2,2 h 14 c 1.105,0 2,-0.9 2,-2 V 16 12 8 L 15,2 Z' /%3E%3Cpath inkscape:connector-curvature='0' id='path12' fill='%233498db' d='M 5,1 C 3.8954,1 3,1.9 3,3 v 8 4 6 c 0,1.1 0.8954,2 2,2 h 14 c 1.105,0 2,-0.9 2,-2 V 15 11 7 L 15,1 Z' /%3E%3Cpath inkscape:connector-curvature='0' id='path14' fill='%232980b9' d='M 21,7 15,1 v 4 c 0,1.1 0.895,2 2,2 z' /%3E%3C/g%3E%3C/svg%3E%0A");
```
Comment (corentinbettiol) [inline filer/private/sass/components/_drag-and-drop.scss:369]: We might replace this "heavy" svg code by this lighter one:

```html
data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath fill='%232980b9' d='M5 2c-1.105 0-2 .9-2 2v18c0 1.1.895 2 2 2h14c1.105 0 2-.9 2-2V8l-6-6z'/%3E%3Cpath fill='%233498db' d='M5 1c-1.105 0-2 .9-2 2v18c0 1.1.895 2 2 2h14c1.105 0 2-.9 2-2V7l-6-6z'/%3E%3Cpath fill='%232980b9' d='m21 7-6-6v4c0 1.1.895 2 2 2z'/%3E%3C/svg%3E
```

(I used svgoptimizer.com to reduce the file size & [this project](https://yoksel.github.io/url-encoder/) to encode it for browsers)

**Pull request merged**